src/shared/Routes.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Returns URL for a specified Learning Object PDF
+ *
+ * @export
+ * @param {string} learningObjectId Id of the Learning Object to fetch the PDF for
+ */
+export function generatePDFURL(learningObjectId: string): string {
+    return `${process.env.LEARNING_OBJECT_SERVICE_URI}/${learningObjectId}/pdf`;
```

Comment:
We should make this more RESTful
```suggestion
    return `${process.env.LEARNING_OBJECT_SERVICE_URI}/users/${authorUsername}/learning-objects/${learningObjectId}/pdf`;
```